Shared/Reader/ReaderSettingsView.swift

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Textile
+
+struct ReaderSettingsView: View {
+    private enum Constants {
+        static let allowedAdjustments = -6...6
+        static let adjustmentStep = 2
+        static let allowedFontFamilies: [FontDescriptor.Family] = [.graphik, .blanco]
+    }
+    
+    @Environment(\.presentationMode)
+    private var presentationMode
+    
+    @EnvironmentObject
+    private var settings: ReaderSettings
+    
+    var body: some View {
+        NavigationView {
+            Form {
+                Section(header: Text("Display Settings")) {
+                    Picker("Font", selection: $settings.fontFamily) {
+                        ForEach(0..<Constants.allowedFontFamilies.count) {
+                            Text(Constants.allowedFontFamilies[$0].name)
+                                .tag(Constants.allowedFontFamilies[$0])
+                        }
+                    }
+                    
+                    Stepper("Font Size",
+                            value: $settings.fontSizeAdjustment,
+                            in: Constants.allowedAdjustments,
+                            step: Constants.adjustmentStep)
+                }
+                
+                Section(header: Text("Preview")) {
+                    Text(
```

Comment:
I like to make a special exception for chopped down method calls when the first param is an array or dictionary literal. Keep the opening square bracket on the same line as the opening paren.

Example:

```swift
myFunctionThatTakesAnArray([
  "item 1",
  "item 2",
  "item 3"
]) // closing bracket and paren on the same line
```

Reply:
👍 I think the other thing to do here is extract the preview text generation into its own thing as well. I feel like having a length single argument isn't the _nicest_ thing. I'll give that a shot and see how things get cleaned up!